src/toil/cwl/cwltoil.py

```diff
@@ -192,6 +192,7 @@ def __init__(self, tool, cwljob):
         #super(CWLJob, self).__init__()
         self.cwltool = tool
         self.cwljob = cwljob
+        self.executor_options = kwargs
```

Comment:
For backwards compatibility, you would need to ensure that `use_container` defaults to True if it is absent from the dictionary. But wait a minute. Actually, I would only default use_container if kwargs is empty:

```
self.executor_options = kwargs or dict( use_container, True )
```

But I don't know if you want to be backwards compatible. @tetron?